docs/content/stable/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md

```diff
@@ -29,17 +30,88 @@ curl --request PUT \
+
+### Built-in roles
+
+The following built-in roles are available:
+
+- **Super Admin** is the first user that is created during installation. This role has the highest level of privilege and allows all read and write actions on all YugabyteDB Anywhere resources. There can be only one Super Admin. Super Admin can perform the following:
+
+  - Manage all resources, including universes, nodes, backup, restore, and cloud providers.
+  - Manage the user access control by creating and managing users.
+
+  For more information, see [Create admin user](../../configure-yugabyte-platform/create-admin-user/).
+
+- **Admin** has privileges that are similar to the Super Admin, except that Admin cannot manage global scope artifacts and actions, such as runtime configuration settings and LDAP authentication.
+
+- **Backup Admin** has access to backup-related tasks, such as the following:
+
+  - Manage database backups and restore operations.
+  - Create new backups.
+  - Delete any existing backup or backup schedule.
+  - Edit existing backups.
+
+  Backup Admin has View permissions for all other resources.
+
+- **Read-only** access level provides view permissions for the UI and API.
+
+- **Connect-only** access level allows users to sign in and access their user profile only. This role is assigned to users who are not explicitly assigned a role.
+
+You can't delete or edit built-in roles.
+
+## Manage users
+
+How you add and modify users varies depending on whether you have enabled fine-grained RBAC. You can only assign, create, and modify custom roles if you have enabled fine-grained RBAC.
+
+<ul class="nav nav-tabs-alt nav-tabs-yb custom-tabs">
+  <li>
+    <a href="#classic" class="nav-link active" id="classic-tab" data-toggle="tab"
+      role="tab" aria-controls="classic" aria-selected="true">
+      Classic
+    </a>
+  </li>
+  <li>
+    <a href="#finegrain" class="nav-link" id="finegrain-tab" data-toggle="tab"
+      role="tab" aria-controls="finegrain" aria-selected="false">
+      Fine grained
+    </a>
+  </li>
+</ul>
+<div class="tab-content">
+  <div id="classic" class="tab-pane fade show active" role="tabpanel" aria-labelledby="classic-tab">
+
+As a Super Admin or Admin, you can invite new users and manage existing users for your tenant.
+
+You can invite new users to your tenant as follows:
+
+1. Navigate to **Admin > User Management > Users** and click **Add User**.
 
-### Create, modify, and delete users
+1. Enter the user's email.
+
+1. Enter a password for the user to sign in with.
+
+1. Select a [built-in role](#built-in-roles) for the user.
+
+1. Click **Submit**.
+
+To modify a user role or delete the user, navigate to **Admin > User Management > Users**. Click **Actions** that corresponds to the specific user, and then select either **Edit User Role** or **Delete User**.
+
+  </div>
+
+  <div id="finegrain" class="tab-pane fade" role="tabpanel" aria-labelledby="finegrain-tab">
 
```

Comment:
felt that you can mention about the version part again that fine-grained RBAC is available from 2.20.1 just in brackets where you the tabs maybe.